app/controllers/admins_controller.rb

```diff
@@ -57,4 +64,16 @@ def delete_p
         procedure.destroy
         redirect_to admins_path
     end
+    
+    def show_items
+      @items = Item.all
+    end
+    
+    def show_procedures
+      @procedures = Procedure.all
+    end
+    
+    def show_facilities
+      @facilities = Facility.all
+    end
```

Comment:
Why are these methods and associated views defined here instead of just using the `index` methods for those models?

Reply:
The index is being used for pending procedures and items but facilities doesn't have a pending or approved status.

Reply:
I was asking why you're not using the `index` methods from the actual `facility`, `item`, and `procedure` controllers.